cmd/sonobuoy/app/args.go

```diff
@@ -87,12 +96,16 @@ func AddE2EConfigFlags(flags *pflag.FlagSet) *pflag.FlagSet {
 		e2eSkipFlag, defaultMode.E2EConfig.Skip,
 		"Specify the E2E_SKIP flag to the conformance tests. Overrides --mode.",
 	)
+	e2eFlags.String(
```

Comment:
kind of want to hide this for power CI users only, b/c folks can destroy their clusters.

Reply:
Marked flag as hidden.